src/SDKs/CognitiveServices/dataPlane/AnomalyDetector/AnomalyDetector/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Cognitive Services AnomalyDetector SDK")]
+[assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services AnomalyDetector APIs.")]
+
+[assembly: AssemblyVersion("2.0.0.0")]
+[assembly: AssemblyFileVersion("2.0.0.0")]
```

Comment:
any reason use 2.* instead of 1.* or 0.*?

Reply:
I thought this should align with the version in swagger file.

Reply:
then make this to be 1.0.0.0